generative_model_training/05_plot_labels_sampling_performance.py

```diff
@@ -0,0 +1,511 @@
+        ymax="aupr_upper_ci",
+        group="label_source",
+        color="label_source",
+    )
+    + p9.geom_point()
+    + p9.geom_line()
+    + p9.geom_errorbar()
+    + p9.facet_wrap("~ prediction_label")
+    + p9.theme_bw()
+    + p9.theme(figure_size=(10, 10))
+    + p9.scale_color_brewer(type="qual", palette=2)
+)
+print(g)
+
+# ## Area Under Receiver Operating Curve (AUROC)
+
+# ### Tune Set
+
+g = (
+    p9.ggplot(
+        performance_ci_df
+        >> ply.query("data_source=='full_text'")
+        >> ply.query("model=='tune'")
+    )
+    + p9.aes(
+        x="lf_num",
+        y="auroc_mean",
+        ymin="auroc_lower_ci",
+        ymax="auroc_upper_ci",
+        group="label_source",
+        color="label_source",
+    )
+    + p9.geom_point()
+    + p9.geom_line()
+    + p9.geom_errorbar()
+    + p9.facet_wrap("~ prediction_label")
+    + p9.theme_bw()
+    + p9.theme(figure_size=(10, 10))
+    + p9.scale_color_brewer(type="qual", palette=2)
+)
+print(g)
+
+# ### Test Set
+
+g = (
+    p9.ggplot(
+        performance_ci_df
+        >> ply.query("data_source=='full_text'")
+        >> ply.query("model=='test'")
+    )
+    + p9.aes(
+        x="lf_num",
+        y="auroc_mean",
+        ymin="auroc_lower_ci",
+        ymax="auroc_upper_ci",
+        group="label_source",
+        color="label_source",
+    )
+    + p9.geom_point()
+    + p9.geom_line()
+    + p9.geom_errorbar()
+    + p9.facet_wrap("~ prediction_label")
+    + p9.theme_bw()
+    + p9.theme(figure_size=(10, 10))
+    + p9.scale_color_brewer(type="qual", palette=2)
+)
+print(g)
+
+# ## Binary Cross Entropy Loss (BCE)
+
+# ### Tune Set
+
+g = (
+    p9.ggplot(
+        performance_ci_df
+        >> ply.query("data_source=='full_text'")
+        >> ply.query("model=='tune'")
+    )
+    + p9.aes(
+        x="lf_num",
+        y="bce_mean",
+        ymin="bce_lower_ci",
+        ymax="bce_upper_ci",
+        group="label_source",
+        color="label_source",
+    )
+    + p9.geom_point()
+    + p9.geom_line()
+    + p9.geom_errorbar()
+    + p9.facet_wrap("~ prediction_label")
+    + p9.theme_bw()
+    + p9.theme(figure_size=(10, 10))
+    + p9.scale_color_brewer(type="qual", palette=2)
+)
+print(g)
+
+# ### Test Set
+
+g = (
+    p9.ggplot(
+        performance_ci_df
+        >> ply.query("data_source=='full_text'")
+        >> ply.query("model=='test'")
+    )
+    + p9.aes(
+        x="lf_num",
+        y="bce_mean",
+        ymin="bce_lower_ci",
+        ymax="bce_upper_ci",
+        group="label_source",
+        color="label_source",
+    )
+    + p9.geom_point()
+    + p9.geom_line()
+    + p9.geom_errorbar()
+    + p9.facet_wrap("~ prediction_label")
+    + p9.theme_bw()
+    + p9.theme(figure_size=(10, 10))
+    + p9.scale_color_brewer(type="qual", palette=2)
+)
+print(g)
+
+# # Abstracts vs Full Text
+
+g = (
+    p9.ggplot(
+        performance_ci_df
+        >> ply.query("prediction_label==label_source")
+        >> ply.query("model=='test'")
```

Comment:
Is GiG missing a line, or are the abstract and full text performances the same?

Reply:
>Is GiG missing a line, or are the abstract and full text performances the same?

If this is the full text plots GiG can't be determined as I can't load the full labeled dataset into memory. (> 64GB and my system only allows for 64GB). Plus, to make matters worse the snorkel package doesn't allow for iterative training without significant changes to their code. Since full text didn't have much improved performance, there isn't enough reason to walk down this path.